rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/webapp/RestRuntime.java

```diff
@@ -140,7 +140,6 @@ public void stop() {
         sessionCleaner.stop();
 
         SchedulerStateListener.getInstance().kill();
-        RMStateCaching.kill();
```

Comment:
why we removed this ?

Reply:
Before RMStateCaching was launching another thread which every %timeperiod% request the latest state from the RMProxyListenerInterface. So RMStateCaching was caching the latest value (now I thing it is better to change it is name). I removed this additional background thread. So now, when rm portal sends request, it take the latest value directly. So no need to kill, because there is nothing to kill